src/components/HarvestingConfiguration/Fields/HarvestingStatusSelect.js

```diff
@@ -7,12 +8,12 @@ import { required } from '../../../util/validate';
 import statusOptions from '../../../util/data/statusOptions';
 import useTranslateLabels from '../../../util/hooks/useTranslateLabels';
 
-const HarvestingStatusSelect = () => {
+const HarvestingStatusSelect = ({ isProviderStatusInactive }) => {
```

Comment:
Please consider renaming the component prop to something like `disabled` or `inactive` so that it better reflects the props intended purpose.